pyproject.toml

```diff
@@ -12,6 +12,7 @@ classifiers = [
     "Intended Audience :: Developers",
     "Operating System :: OS Independent",
     "Programming Language :: Python",
+    "Programming Language :: Python :: Free Threading :: 2 - Beta",
```

Comment:
Why is this "beta"? Why not "stable" or "resilient"?

Reply:
Hi, please refer to this: https://py-free-threading.github.io/porting/#define-and-document-thread-safety-guarantees

1. Alpha - For experimentation and feedback only.
2. Beta - Free threaded usage is supported, but documentation of constraints and limitations may be incomplete.
3. Stable - Supported for production use, multithreaded use is tested, and thread safety issues are clearly documented.
4. Resilient - Fully supported and fully thread safe.

In general, I would not go beyond Beta status before having some feedback that everything works well in production environments.